heat/core/manipulations.py

```diff
@@ -244,6 +245,52 @@ def broadcast_to(x: DNDarray, shape: Tuple[int, ...]) -> DNDarray:
     return broadcasted
 
 
+def collect(arr: DNDarray, target_rank: Optional[int] = 0) -> DNDarray:
+    """
+    A method collecting a distributed tensor to one rank, chosen by the target_rnk variable, and zero by default.
```

Comment:
```suggestion
    A function collecting a distributed DNDarray to one rank, chosen by the target_rank variable.
```

Reply:
We should mention that this function allocates a new DNDarray, and refer to the in-place operation.